### PR TITLE
TaskParameter.contents -> content

### PR DESCRIPTION
**Pull request by buchanae**

### CODE REVIEW

**Comment by prismofeverything** [comment]
+1 This makes more sense to me.

**Comment by briandoconnor** [comment]
Seems fine to me...

File: task_execution.proto [hunk 1]
```diff
@@ -113,7 +113,7 @@ message TaskParameter {
   // UTF-8 encoded
   //
   // If contents is not empty, "url" must be ignored.
```
Comment (geoffjentry) [inline task_execution.proto]: should update the comment :)

**Comment by geoffjentry** [comment]
:+1:

File: task_execution.proto (hunk 1 above)
Comment (buchanae) [reply task_execution.proto]: Good catch. Updated.

**Comment by pditommaso** [comment]
👍

**Pull request merged**